app/screens/InfoScreen.tsx

```diff
@@ -25,7 +33,9 @@ export const InfoScreen: React.FunctionComponent<TabScreenProps<"Info">> = () =>
   useHeader({
     title: translate("infoScreen.title"),
     RightActionComponent: (
-      <Button preset="link" text="Contact" onPress={contactByEmail} style={$emailButton} />
+      <Button preset="link" onPress={contactByEmail} style={$emailButton}>
+        <RNText maxFontSizeMultiplier={1.8}>Contact</RNText>
```

Comment:
I'm not sure if there's a better way constrain the font scaling without changing the default styling on the button and text component.

Reply:
We could look to add a prop like `TextProps` to the `<Button />` maybe and pass it through so we don't have to double import everywhere we use button and need font scaling? `<Screen />` does this with the `ScrollViewProps`.

For example, do you think something like this cleans things up?

```
<Button preset="link" onPress={contactByEmail} style={$emailButton} TextProps={{ maxFontSizeMultiplier: 1.8 }} />
```

and in `Button.tsx`

```
import { TextProps as RNTextProps } from "react-native"
// ...

// adjust interface ButtonProps
TextProps?: RNTextProps

// ...

<Text tx={tx} text={text} txOptions={txOptions} style={$textStyle(state)} {...TextProps}>
  {children}
</Text>
```

Thoughts?

Reply:
This is definitely the way I was also considering since we did it several times with our recent client work. Just wasn't sure if I should add more props to the base components in Ignite. I will go ahead and add the `TextProps` then. Thanks @frankcalise